src/main/kotlin/com/cognifide/gradle/aem/api/AemConfig.kt

```diff
@@ -555,6 +556,28 @@ class AemConfig(
             "*_x0040_TypeHint"
     )
 
+    /**
+     * Define here mixin types that will be skipped when pulling JCR content from AEM instance.
+     */
+    @Input
+    var cleanSkipMixinTypes: MutableList<String> = mutableListOf(
+            "cq:ReplicationStatus",
+            "mix:versionable"
+    )
+
+    /**
+     * Define hook method for customizing properties clean up.
+     */
+    @Internal
+    @get:JsonIgnore
+    var cleanLineProcess: (VltCleaner, File, String) -> String = { vltCleaner, file, line -> vltCleaner.normalizeLine(file, line) }
+
+    /**
+     * Turn on/off namespace normalization after properties clean up.
+     */
+    @Input
+    var cleanNamespaces: Boolean = true
```

Comment:
I am not sure if we should introduce so many properties in config, instead configuring all in lambda ;)

`cleanLineProcess = {  chain().skipProperties(['a','b','b']).cleanNamespaces().cleanMixinTypes(['a', 'b', 'c']) }`

personally I prefer that instead of growing and growing AEM config

Reply:
for now it could stay as is